### PR TITLE
(Refactor) refactored flight API

### PR DESCRIPTION
**Pull request by joshjnunez**
Included getCoordinates function, and others to reuse code
Cleaned up API call 
Made data dynamic

### CODE REVIEW

**Review by gjones2019 — APPROVED** [review]

**Review by jldaggs88 — APPROVED** [review]

File: client/components/Flights.js [hunk 1]
```diff
@@ -5,34 +5,33 @@ import axios from 'axios';
...
+  // const handleChange = (response) => {
+  //   setTrip(response);
+  // };
 
   const handleChangeFlight = (response) => {
     setFlightData(response);
   };
 
   useEffect(() => {
+    // axios
```
Comment (jstrzesz) [inline client/components/Flights.js:19]: is this code going to be used again in future? If not, please remove

File: server/flights.js [hunk 2]
```diff
@@ -1,19 +1,38 @@
...
-const getFlightsInfo = async () => {
+const getFlightsInfo = async (tripInfo) => {
   // input: object with trip ID, destination
   // output: array of objects - flight info
   // array of objects - destinations: airport codes
 
+  // axios.post('/getFullTrip', { id: tripInfo.id }, (res) => {
+  //   console.info(res);
+  // });
+  console.info(tripInfo);
```
Comment (jstrzesz) [inline server/flights.js]: remove

File: server/index.js [hunk 3]
```diff
@@ -201,7 +208,8 @@ app.post('/getAllTrips', authCheck, (req, res) => {
 });
 
 app.post('/getFlights', authCheck, (req, res) => {
-  getFlights(req, res);
+  console.info('!!!!!', req.body, '!!!!!!!!!!');
```
Comment (jstrzesz) [inline server/index.js:212]: remove

File: server/queries.js [hunk 4]
```diff
@@ -445,18 +445,18 @@ const getFlights = async (req, res) => {
 };
 
 const getFullTrip = async (req, res) => {
-  console.info(req, 'REQ');
+  // console.info(req, 'REQ');
```
Comment (jstrzesz) [inline server/queries.js:448]: remove

File: server/queries.js [hunk 5]
```diff
@@ -445,18 +445,18 @@ const getFlights = async (req, res) => {
...
   const trip = await Trip.findOne({
     where: { id: req.body.id },
   });
-  console.info('trip found', trip);
+  // console.info('trip found', trip);
```
Comment (jstrzesz) [inline server/queries.js:452]: remove

**Pull request closed without merging**